core/helpers/assertions.ts

```diff
@@ -0,0 +1,67 @@
+import BigNumber from '../src/bignumber';
+import getContract, { getErc20Contract } from '../src/contracts';
+import { CollateralType } from '../src/types';
+import { ethers } from 'ethers';
+import { WAD_NUMBER_OF_DIGITS } from '../src/constants/UNITS';
+
+export const assertVatCollateralBalance = async (
```

Comment:
Let's not create a new `assertions` file with a single-purpose functions. Instead, we can:
- create functions (if they not already exist) that fetch and return what we need
- do assertion directly in the function that does modification (or in the test)